actors/builtin/miner/miner_state.go

```diff
@@ -1068,6 +1068,11 @@ func deleteMany(arr *adt.Array, keys []uint64) error {
 	return nil
 }
 
+func quantizeUpWithOffset(e abi.ChainEpoch, unit, offsetSeed abi.ChainEpoch) abi.ChainEpoch {
+	offset := offsetSeed % unit
+	return quantizeUp(e, unit) + offset
```

Comment:
This is the only caller of `quantizeUp`. Please inline it here so that no-one else introduces the same kind of problem by using `quantizeUp` directly.